README.md

```diff
@@ -74,6 +76,21 @@ printer
   .catch(console.error);
 ```
 
+To scale the PDF to fit into the printable area of the paper on both Windows and Unix operating systems:
+
+```javascript
+const options = {
+  printer: "Zebra",
+  unix: ["-o fit-to-page"],
+  win32: ['"-print-settings "fit"']
```

Comment:
I think that the quote before the `-print-settings` is unnecessary in this example `'"-print-settings "fit"'`